### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by brmatthy**
This PR creates a cleaner file structure of the webpage.
It also fixes the issues we had with the map.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -66,32 +26,196 @@ import {Query, getMonths, getYears, getCategories} from "./components/queries.js
...
 import {svg} from "npm:htl";
+import * as d3 from "npm:d3";
 ```
 
 ```js
 // load data
 const crimeData = await loadCrimeData();
 const geoData = await loadGeometryData();
+geoData.features.forEach((g) => {
+    // coordinates are inside out, so reverse them
```
Comment (matsvbelle) [inline docs/index.md:37]: Maybe save this already reversed in the json file? Do we ever use the normal coordinates?

File: docs/index.md [hunk 2]
```diff
@@ -108,16 +232,7 @@ const resultPerMonth = new Query(crimeData).filterByCategory(category).groupByYe
...
-```js
-const years = getYears().map(String);
-const year = view(Inputs.select(years, {value: "2018", label: "Selecteer jaar:"}));
-const category_parallel = view(Inputs.select(categories, {value: "Graffiti", label: "Selecteer categorie:"}));
-```
 
-```js
-display(parallel(crimeData, year, category_parallel));
-```
 ## Amount of crimes
```
Comment (matsvbelle) [inline docs/index.md:236]: Maybe we can already remove all these old bar charts? They are slow, clunky and if necessary we can quickly add better ones using observable plot. To start restructuring, I would remove them.

File: docs/index.md [hunk 3]
```diff
@@ -240,3 +355,7 @@ amountOfCrimesPerYear.setOption({
 });
 ```
 
+## Gentse feesten
```
Comment (matsvbelle) [inline docs/index.md:360]: Add another TODO with the data split into severe and less severe crimes.

**Review by matsvbelle — APPROVED** [review]
Looks amazing!!! I added a few minor comments but I think you can also merge without the changes.

**Review by warreprovoost — APPROVED** [review]

File: docs/index.md (hunk 2 above)
Comment (brmatthy) [reply docs/index.md:236]: I would replace them once the replacements are done. This way we have an example.

File: docs/index.md (hunk 1 above)
Comment (brmatthy) [reply docs/index.md:37]: I don't think we do, but it's really not computationally intensive. there are only 25 regions

**Pull request merged**